frontend/package.json

```
@@ -26,7 +26,8 @@
     "verify": "yarn build && yarn lint && yarn test"
   },
   "dependencies": {
-    "@openshift/dynamic-plugin-sdk": "^1.0.0-alpha16",
+    "@openshift/dynamic-plugin-sdk": "^1.0.0",
+    "@openshift/dynamic-plugin-sdk-extensions": "^1.0.0",
     "@openshift/dynamic-plugin-sdk-utils": "^1.0.0-alpha19",
```

For the new HAC infra functionality, we need this bumped up to version 1
```suggestion
    "@openshift/dynamic-plugin-sdk-utils": "^1.0.0",
```

along with the corresponding updated yarn.lock file

---

I was actually testing against the wrong PR.  Ignore this comment.  These are not the droids you are looking for.  Move along.